src/app/inventory/ItemIcon.m.scss

```diff
@@ -79,10 +79,11 @@ $commonBg: #366f42;
   background-image: url('../../images/exotic-masterwork.png');
 }
 .deepsightBorder {
-  background-image: url('../../images/deepsight.png');
+  border: 2px solid $deepsight-border-color;
```

Comment:
```suggestion
  border: 2 * $item-border-width solid $deepsight-border-color;
```

Reply:
Trial and error reduced until wasn't able to replicate bug any more
You wouldn't be able to tell difference even if it was removed all together
It's -0.4 and not removed just in case border size would get bigger one day